apps/oxlint/src/command/lint.rs

```diff
@@ -122,6 +122,31 @@ pub struct FixOptions {
     /// Fix as many issues as possible. Only unfixed issues are reported in the output
     #[bpaf(switch)]
     pub fix: bool,
+    /// Apply all safe fixes and more-risky suggestions. Overrides `--fix`.
+    #[bpaf(switch)]
+    pub fix_suggestions: bool,
```

Comment:
In my personal view, when I use`fix_suggestions` without taking a look at documents, my intuition tells me it will fix only suggestions. It's convenient to apply multi kind fixes in one command, but people might want to apply suggestions alone.

Reply:
That makes sense; how about a fix level, kind of like -O3?

Reply:
LGTM